src/test/java/org/skriptlang/skript/test/tests/utils/DateTest.java

```diff
@@ -0,0 +1,30 @@
+package org.skriptlang.skript.test.tests.utils;
+
+import ch.njol.skript.util.Date;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class DateTest {
+
+	@Test
+	public void testNow() {
+		Date now = Date.now();
+		assertEquals(System.currentTimeMillis(), now.getTime());
```

Comment:
Be careful with these because it's technically possible for the system clock to tick during a method (if the processor switches between non-critical tasks). It's very unlikely though so you should be okay.